### PR TITLE
New code examples

### PR DESCRIPTION
**Pull request by AnashOommen**

### CODE REVIEW

File: examples/Misc/GetAllVideosAndImages.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
+﻿// Copyright 2019 Google LLC
```
Comment (CoryLiseno) [inline examples/Misc/GetAllVideosAndImages.cs]: 2020

File: examples/AdvancedOperations/AddSmartDisplayAd.cs [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+            // Create the operation.
+            CampaignOperation operation = new CampaignOperation() { Create = campaign };
+
+            // Add the campaign.
+            MutateCampaignsResponse response = campaignService.MutateCampaigns(
+                customerId.ToString(), new[] { operation });
+
+            string campaignResourceName = response.Results.First().ResourceName;
+
+            // Print out some information about the added ad group.
```
Comment (sundquist) [inline examples/AdvancedOperations/AddSmartDisplayAd.cs]: I think this should be campaign and not ad group based on the context.

File: examples/Misc/GetAllVideosAndImages.cs (hunk 1 above)
Comment (sundquist) [reply examples/Misc/GetAllVideosAndImages.cs]: +1 Across all the files that are newly added in this commit, please.

Comment (AnashOommen) [reply examples/Misc/GetAllVideosAndImages.cs]: Done.

File: examples/AdvancedOperations/AddSmartDisplayAd.cs (hunk 2 above)
Comment (AnashOommen) [reply examples/AdvancedOperations/AddSmartDisplayAd.cs]: Done everywhere.

**Pull request merged**